compiler/haskell-ide-core/src/Development/IDE/LSP/LanguageServer.hs

```diff
@@ -0,0 +1,181 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I’d prefer if we could only duplicate this module. The rest should be reusable as far as I can tell and duplicating everything is going to create a mess.